### ISSUE TITLE
Improve documentation for the first release

### ISSUE THREAD

**Issue by habernal** [dkpro/dkpro-c4corpus#2]
- add some typical use-case scenarios
- describe reproducibility of the reported results
  - [ ] Separate issue #4 
- [x] how to access the corpus from AWS

**Comment by habernal** [comment]
Depends on #4

**Comment by habernal** [comment]
Closing for now - we have several use-cases and quite a detailed description of the pipeline and how to access the data.
